website/docs/language/expressions/custom-conditions.mdx

```diff
@@ -127,10 +127,10 @@ resource "aws_instance" "example" {
       error_message = "The selected AMI must be for the x86_64 architecture."
     }
 
-    # The EC2 instance must be allocated a private DNS hostname.
+    # The EC2 instance must be allocated a public DNS hostname.
     postcondition {
-      condition     = self.private_dns != ""
-      error_message = "EC2 instance must be in a VPC that has private DNS hostnames enabled."
+      condition    = self.public_dns != ""
```

Comment:
```suggestion
      condition     = self.public_dns != ""
```

Equals signs should line up (`terraform fmt` will make them do so)

Reply:
thank you!!